webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/AgentTypes.scala

```diff
@@ -176,44 +180,36 @@ object AgentInfoSerialisation {
       )
   }
 
-  def parseSecurityToken(agentType : AgentType, tokenJson: JValue, tokenDefault : Option[String]) : Box[SecurityToken]= {
-    import net.liftweb.json.compactRender
+  def parseSecurityToken(agentType : AgentType, tokenJson: io.circe.JsonObject, tokenDefault : Option[String]) : Box[SecurityToken]= {
```

Comment:
i'm pretty sure we could do a decoder here, but with the corner cases, i'm not sure it's really possible